src/generic-components/EmployeeDetails.tsx

```diff
@@ -0,0 +1,139 @@
+import * as React from 'react'; 
+import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
+import { View, Image,Text,TouchableOpacity } from 'react-native';
+import styled from 'styled-components/native';
+import Styles from '../CustomReportsPage/style/Styles';
+import { faChevronLeft, faChevronRight } from '@fortawesome/free-solid-svg-icons';
+
+const Header = styled.View`
+    display: flex;
+    flex-direction: row;
+`;
+
+const ContainerView = styled.View`

```

Comment:
As a naming convention I would avoid appending `View` to the end of your styled components styles. This allows us to easily replace any styled elements with an appropriately named component if we feel it necessary.